bin/check_shell_scripts.sh

```diff
@@ -47,3 +47,8 @@ SHEBANG_FILES=$( \
 
 echo "${SH_FILES}" "${SHEBANG_FILES}" \
     | xargs shellcheck
+
+#
+# Execute bash shell unit tests script
+#
+./tests/scripts/scripts_test.sh
```

Comment:
Personally I think this is okay, but I guess this shellcheck was used for lint.

https://github.com/istio/istio/blob/master/Makefile#L446, add a test target to the `TEST_OBJ` or existing `common_test` might fit your intent better (functionality test), rather than style.

Reply:
TEST_OBJ eventually will be part of `make test`, and that will run on prow & circle ci.

Reply:
done